javascript/chrome-driver/atoms.js

```diff
@@ -259,9 +259,17 @@ webdriver.chrome.isElementClickable = function(elem, coord) {
     }
     parentElemIter = parentElemIter.parentNode;
   }
+  var elemHTML = elem.outerHTML;
+  if (elem.hasChildNodes()) {
+    var inner = elem.innerHTML;
```

Comment:
I copied the whole block from line 244. Does your reduction apply to that code block as well?